conda/recipes/rmm/meta.yaml

```diff
@@ -40,22 +40,35 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    - cuda-python 11.7.1
```

Comment:
Could we use [PEP 440 compatible constraints](https://peps.python.org/pep-0440/#version-specifiers) here? That will help us in the long run with https://github.com/rapidsai/dependency-file-generator/issues/42. It is also helpful for those less familiar with conda specs since it's not immediately clear whether the lack of any equality/inequality sign indicates strict equality, SemVer compatible versions, flexibility on the later parts of the release specifier (e.g. is 11.7.1.1 OK?), etc.

Reply:
I'm not entirely sure what this means. Add `=`? Add `==`?

Reply:
Conda specifies its package matching rules here:
https://conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

Reply:
We'd need to review _every_ specifier if you want to do something consistent here... and is conda's matching specification even compliant with PEP 440? Maybe better to file an issue and do this later?

Reply:
Done.